library/src/components/Extensions.tsx

```diff
@@ -1,34 +1,112 @@
 /* eslint-disable @typescript-eslint/no-unsafe-return */
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 
-import React from 'react';
+import React, { useContext, useState } from 'react';
 
 import { Schema } from './Schema';
 
 import { SchemaHelpers } from '../helpers';
+import { AsyncAPIDocumentInterface, BaseModel } from '@asyncapi/parser';
+import { useConfig, useSpec } from '../contexts';
+import { CollapseButton } from './CollapseButton';
 
 interface Props {
   name?: string;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   item: any;
 }
 
+export interface ExtensionComponentProps<V = any> {
+  propertyName: string;
+  propertyValue: V;
+  document: AsyncAPIDocumentInterface;
+  parent: BaseModel;
+}
+
+const SchemaContext = React.createContext({
+  reverse: false,
+});
```

Comment:
You don't use that context in code (only read the context value), so it can be removed.

```suggestion
```